Resources/Prototypes/_LostParadise/Entities/Objects/Weapons/Guns/tags.yml

```
@@ -11,4 +11,7 @@
   id: LPPMagazineMars
 
 - type: Tag
-  id: LPPMagazineGarand
\ No newline at end of file
+  id: LPPMagazineGarand
+
+- type: Tag
+  id: CartridgeMinigun
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)